SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -814,20 +816,20 @@ void USpatialSender::TrackRPC(AActor* Actor, UFunction* Function, const RPCPaylo
 
 bool USpatialSender::WillHaveAuthorityOverActor(AActor* TargetActor, Worker_EntityId TargetEntity)
 {
-	bool WillHaveAuthorityOverActor = true;
+	bool bWillHaveAuthorityOverActor = true;
 
 	if (NetDriver->VirtualWorkerTranslator != nullptr)
 	{
 		if (const SpatialGDK::AuthorityIntent* AuthorityIntentComponent = StaticComponentView->GetComponentData<SpatialGDK::AuthorityIntent>(TargetEntity))
 		{
 			if (AuthorityIntentComponent->VirtualWorkerId != NetDriver->VirtualWorkerTranslator->GetLocalVirtualWorkerId())
 			{
-				WillHaveAuthorityOverActor = false;
+				bWillHaveAuthorityOverActor = false;
 			}
 		}
 	}
 
-	return WillHaveAuthorityOverActor;
+	return bWillHaveAuthorityOverActor;
 }
```

Comment:
I think this whole method can be nuked, it existed for the time when we briefly set Actor role to SimulatedProxy when the entity was being created - this isn't the case anymore. Should be able to replace all occurrences of this function called with `Actor->HasAuthority()` - would you mind drive-by fixing this?

Reply:
Let me know if you think the changes I have made make sense. We might be able to remove the new TargetActor->HasAuthority() check as it might be redundant after NetDriver->StaticComponentView->HasAuthority(EntityId, ComponentId).